chain-signatures/node/src/mesh/mod.rs

```diff
@@ -110,8 +121,31 @@ impl Mesh {
     }
 
     /// Ping the active participants such that we can see who is alive.
-    pub async fn ping(&mut self) {
+    async fn ping(&mut self) {
         self.active_participants = self.connections.ping().await;
         self.active_potential_participants = self.connections.ping_potential().await;
     }
+
+    pub async fn run(
+        mut self,
+        contract_state: Arc<RwLock<Option<ProtocolState>>>,
+        mesh_state: Arc<RwLock<MeshState>>,
+    ) -> anyhow::Result<()> {
+        let mut last_pinged = Instant::now();
+        loop {
+            if last_pinged.elapsed() > Duration::from_millis(300) {
```

Comment:
Is 300ms what we have now? That can create an additional load on RPC. My expectation was ~1s

Reply:
yep

Reply:
Yes, but how many elapsed checks will you do during this 300ms?

Reply:
ah makes sense